src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -193,6 +193,23 @@ export class SearchMenu extends React.Component {
     window.location.assign(searchUrl);
   };
 
+  formatSuggestion = suggestion => {
+    if (!suggestion || !this.state.userInput) {
+      return suggestion;
+    }
+    const lowerSuggestion = suggestion?.toLowerCase();
+    const lowerQuery = this.state.userInput?.toLowerCase();
+    if (lowerSuggestion.includes(lowerQuery)) {
+      return (
+        <>
+          {this.state.userInput}
+          <strong>{lowerSuggestion.replace(lowerQuery, '')}</strong>
+        </>
+      );
+    }
+    return <strong>{lowerSuggestion}</strong>;
+  };
+
```

Comment:
Seems like there should be a unit test for this function.

Reply:
I have a unit test that is testing the formatting, but I will add an additional test for the function specifically

Reply:
If you feel that the existing test provides sufficient coverage, feel free to ignore my suggestion. 😄

Reply:
Based on this, I'm guessing we'll never show a capital letter in the typeahead?

Reply:
correct, the typeahead suggestions we receive from search.gov are supposed to be always lower case. My intention with this implementation is to preserve a user input (if they have a capital) while using the to lower of it for determining what should be bolded. I have the toLower on the suggestion just in case they have some suggestions that break the "only lower case" rule they have set.